values.yaml

```diff
@@ -0,0 +1,35 @@
+redis:
+  enabled: true
+  auth:
+    enabled: false
+  architecture: standalone
+  master:
+    persistence:
+      enabled: true
+      size: 1Gi  # Adjust the size as needed
+
+onetimesecret:
+  image: 
+    registry: docker.io
+    repository: onetimesecret/onetimesecret
+   #tag:latest  ##This will override the .Chart.AppVersion
```

Comment:
There might be a better way to do this. Just looking at another helm chart:

https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/templates/argocd-repo-server/deployment.yaml#L72

https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/templates/_common.tpl#L37-L39

Reply:
we have already defined same helper function as Argocd in helper. Just needed to remove comment for tag from `values.yaml` and keep it null.